apps/examples/testcase/ta_tc/media/itc/itc_media_fileoutputdatasource.cpp

```diff
@@ -90,12 +95,15 @@ static void itc_media_FileOutputDataSource_getPcmFormat_setPcmFormat_p(void)
 {
 	media::stream::FileOutputDataSource source(file_path);
 	TC_ASSERT_EQ("open", source.open(), 1);
-	TC_ASSERT_EQ("getPcmFormat", source.getPcmFormat(), MEDIA_FORMAT_TYPE);
-	source.setPcmFormat(1);
-	TC_ASSERT_EQ("getPcmFormat", source.getPcmFormat(), 1);
-	source.setPcmFormat(2);
-	TC_ASSERT_EQ("getPcmFormat", source.getPcmFormat(), 2);
+	TC_ASSERT_EQ("getPcmFormat", source.getPcmFormat(), media::AUDIO_FORMAT_TYPE_S16_LE);
+	source.setPcmFormat(media::AUDIO_FORMAT_TYPE_S8);
+	TC_ASSERT_EQ("getPcmFormat", source.getPcmFormat(), media::AUDIO_FORMAT_TYPE_S8);
```

Comment:
Use TC_ASSERT_EQ_CLEANUP macro to take care of close in case of failure.
Check it for all such occurences.

Reply:
changes done and code also updated in gerrit